server/webapp/WEB-INF/rails.new/app/assets/stylesheets/css/application-old.scss

```diff
@@ -1396,6 +1396,7 @@ a:visited#link-to-this-page {
     font-size: 11px;
     /*text-decoration: underline;*/
     float: right;
+    color: white;

```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces